ibc/module.go

```diff
@@ -1,34 +1,28 @@
 package ibc
 
 import (
-	"encoding/hex"
-	"fmt"
-	"sync"
+	"context"
 
-	"github.com/pokt-network/pocket/ibc/store"
-	ibcTypes "github.com/pokt-network/pocket/ibc/types"
+	"github.com/pokt-network/pocket/ibc/host"
 	"github.com/pokt-network/pocket/logger"
 	"github.com/pokt-network/pocket/runtime/configs"
-	"github.com/pokt-network/pocket/shared/codec"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
-	"github.com/pokt-network/pocket/shared/crypto"
 	"github.com/pokt-network/pocket/shared/modules"
 	"github.com/pokt-network/pocket/shared/modules/base_modules"
-	"google.golang.org/protobuf/types/known/anypb"
 )
 
 var _ modules.IBCModule = &ibcModule{}
 
 type ibcModule struct {
 	base_modules.IntegrableModule
 
-	m sync.Mutex
-
 	cfg    *configs.IBCConfig
 	logger *modules.Logger
 
-	// Only a single host is allowed at a time
```

Comment:
Does this comment need to be recaptured somewhere else or is it no longer applicable?